.github/workflows/test.yml

```diff
@@ -35,8 +35,21 @@ jobs:
       - name: Run rustfmt --check
         run: cargo fmt -- --check
 
+  msrv:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      # rust-version from Cargo.toml
+      - name: Install Rust 1.66.0
```

Comment:
perhaps we could use `cargo hack --rust-version` reading `rust-version` from the manifest instead of managing this manually?

```
- uses: taiki-e/install-action@cargo-hack
```

Reply:
Hmm it's possible, but it makes caching a bit more complicated. I'll see if I can do it..

Reply:
Oh, nevermind. The obvious solution actually works fine. Not sure what I expected